### PR TITLE
Programmatically create Beam view.

### PR DESCRIPTION
**Pull request by jasonkuster**
Signed-off-by: Jason Kuster <jasonkuster@google.com>

Be sure to do all of the following to help us incorporate your contribution
quickly and easily:

 - [ ] Make sure the PR title is formatted like:
   `[BEAM-<Jira issue #>] Description of pull request`
 - [ ] Make sure tests pass via `mvn clean verify`.
 - [ ] Replace `<Jira issue #>` in the title with the actual Jira issue
       number, if there is one.
 - [ ] If this contribution is large, please file an Apache
       [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

---

### CODE REVIEW

**Comment by jasonkuster** [comment]
R: @davorbonaci

**Comment by jasonkuster** [comment]
- Just moves how this is defined. My understanding from playing around on my own jenkins is that the A-D view is comprised of other views defined on Jenkins; I wasn't able to find a way to define this view in the context of the A-D view.
- Will do
- My philosophy on this is launch and iterate -- want to make sure we're generating the basic view correctly (control number of things changing), then evaluate fancier options.

File: .test-infra/jenkins/job_beam_ListView_Create.groovy [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        all()
+    }
+    columns {
+        status()
+        weather()
+        name()
+        lastSuccess()
+        lastFailure()
+        lastDuration()
+        buildButton()
```
Comment (tgroh) [inline .test-infra/jenkins/job_beam_ListView_Create.groovy]: No more console or last success description, or are those included other ways

Comment (jasonkuster) [inline .test-infra/jenkins/job_beam_ListView_Create.groovy]: https://jenkinsci.github.io/job-dsl-plugin/#path/listView-columns doesn't list the ability to have the last success description. Have added console.

**Pull request closed without merging**